index.bs

```diff
@@ -717,6 +717,56 @@ is done in a consistent and privacy-friendly way:
      stable id during the session for the given origin, but also be able to deterministically produce
      the same id in subsequent sessions.
 
+<h3 id="wrapper-apis">Wrapper API considerations</h3>
```

Comment:
Slightly more wordy but descriptive title suggetion: "APIs that simply pass-through OS APIs don't typically translate well to the web."